init.lua

```diff
@@ -186,6 +181,8 @@ minetest.register_node("memorandum:letter_written", {
 		local item = sender:get_wielded_item()
 		if item:get_name() == "memorandum:eraser" then
 			local meta = minetest.get_meta(pos)
+			fields.text = fields.text:sub(1, 10000)
+			fields.signed = fields.signed:sub(1, 80)
```

Comment:
```suggestion
			fields.text = (fields.text or ""):sub(1, 10000)
			fields.signed = (fields.signed or ""):sub(1, 80)
```

Reply:
there's a nil check for those values a couple lines up, is this still needed?

Reply:
I think that check is in a different `on_receive_fields` function